File: Examples/VisualBasic/3DScene/FlipCoordinateSystem.vb
Imports System.IO
Imports System.Collections
Imports Aspose.ThreeD
Imports Aspose.ThreeD.Animation
Imports Aspose.ThreeD.Entities
Imports Aspose.ThreeD.Formats

Namespace _3DScene
    Class FlipCoordinateSystem
        Public Shared Sub Run()
            ' ExStart:FlipCoordinateSystem
            ' The path to the documents directory.
            Dim MyDir As String = RunExamples.GetDataDir()
            ' Initialize scene object
            Dim scene As New Scene()
            scene.Open(MyDir & Convert.ToString("camera.3ds"), FileFormat.Discreet3DS)
            MyDir = MyDir & Convert.ToString("FlipCoordinateSystem.obj")
            scene.Save(MyDir, FileFormat.WavefrontOBJ)
            ' ExEnd:FlipCoordinateSystem
            Console.WriteLine(Convert.ToString(vbLf & "Coordinate system has been flipped successfully." & vbLf & "File saved at ") & MyDir)
        End Sub
    End Class
End Namespace

File: Examples/CSharp/3DScene/FlipCoordinateSystem .cs
using System;
using System.IO;
using System.Collections;
using Aspose.ThreeD;
using Aspose.ThreeD.Animation;
using Aspose.ThreeD.Entities;
using Aspose.ThreeD.Formats;

namespace Aspose._3D.Examples.CSharp._3DScene
{
    class FlipCoordinateSystem
    {
        public static void Run()
        {
            // ExStart:FlipCoordinateSystem
            // The path to the documents directory.
            string MyDir = RunExamples.GetDataDir();            
            // Initialize scene object
            Scene scene = new Scene();
            scene.Open(MyDir + "camera.3ds", FileFormat.Discreet3DS);
            MyDir = MyDir + "FlipCoordinateSystem.obj";
            scene.Save(MyDir, FileFormat.WavefrontOBJ);
            // ExEnd:FlipCoordinateSystem
            Console.WriteLine("\nCoordinate system has been flipped successfully.\nFile saved at " + MyDir);
        }
    }
}
